python/py-prompt_toolkit/Portfile

```diff
@@ -1,11 +1,8 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
And why we should add something to **every** file to tell editor which settings to use?